lib/ember-test-helpers/abstract-test-module.js

```diff
@@ -100,12 +104,26 @@ export default Klass.extend({
   },
 
   setupContext(options) {
-    var config = assign({
+    let context = this.getContext();
+
+    merge(context, {
       dispatcher: null,
       inject: {}
-    }, options);
+    });
+    merge(context, options);
+
+    setContext(context);
+    this.context = context;
```

Comment:
ah crap, you're right... missed that the `this` was missing... this is not Java... 😉